### DISCUSSION TITLE
How to create dropdown without lookupEditor and ServiceLookupEditor

### DISCUSSION THREAD

**Discussion by armanmahmoudi** [serenity-is/Serenity#7155] [General]
Hi every one.
I want to create a dropdown menu whitout LookupEditor and ServiceLookupEditor?
can you help me?

**Comment by LSNicholls** [comment]
I am not sure anybody will be able to help you without understanding why you don't want to use LookupEditor or ServiceLookupEditor.  So, explain your reasoning/requirements, and somebody may be able to help you :-)

**Reply by Vinith-kumar47** [reply]
Hi Mam,

I know that this is not the right thread to ask you this.
But I have one doubt in my serenity project and I posted a help in this group (#7177).
If you have time, then please look into my question and help me on this.

Thanks 
Vinith
